### DISCUSSION TITLE
Similar software for windows please

### DISCUSSION THREAD

**Discussion by SpilledSauce** [JunkFood02/Seal#243] [Q&A]
Non-techie this side.
Sorry for the disturbance.
Can somebody recommend me a similar software for windows ?
Thanks in advance

**Comment by ghost** [comment]
https://github.com/ytdl-org/youtube-dl
I recommend this one. Since you are non-techie go for this https://github.com/Tyrrrz/YoutubeDownloader

You can go through this too
https://github.com/JunkFood02/Seal/discussions/210
